### PR TITLE
docs: update Avro scheme example

### PR DESCRIPTION
**Pull request by zamazan4ik**
- remove redundant '"' from JSON

Tested:
- Local Avro serializer craetion with the example

### CODE REVIEW

**Comment by zamazan4ik** [comment]
@neuronull Just a small fix :)

File: website/cue/reference/components/sinks.cue [hunk 1]
```diff
@@ -236,7 +236,7 @@ components: sinks: [Name=string]: {
 													type: string: {
 														examples: [
 															"""
-															["{ "type": "record", "name": "log", "fields": [{ "name": "message", "type": "string" }] }"]
+															[{ "type": "record", "name": "log", "fields": [{ "name": "message", "type": "string" }] }]
```
Comment (jszwedko) [inline website/cue/reference/components/sinks.cue:239]: I think the schema actually does need to be defined as a string containing JSON 🤔

Comment (zamazan4ik) [reply website/cue/reference/components/sinks.cue:239]: If you use the following configuration:
```
encoding.avro.schema = "[\"{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }\"]"
```
and run Vector, you will get the following error:
```
2022-11-08T21:08:38.353507Z ERROR vector::topology: Configuration error. error=Sink "kafka": Failed building Avro serializer: Failed to parse schema from JSON
```

But if you use the following config:
```
encoding.avro.schema = "[{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }]"
```
Vector initializes Avro serializer successfully.

Comment (zamazan4ik) [reply website/cue/reference/components/sinks.cue:239]: If you will check the code, you can find that as a string is treated the whole `encoding.avro.schema` field. However, probably as a valid Avro schema will be the following string:
```
encoding.avro.schema = "{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }"
```
without `[]` brackets.

Comment (jszwedko) [reply website/cue/reference/components/sinks.cue:239]: Doh, yeah, I see, it is double quoted. I missed that this is already in a `"""` block.

**Review by jszwedko — APPROVED** [review]
Thanks!

**Comment by zamazan4ik** [comment]
@jszwedko I guess even outer `[]` brackets should be removed from the sample. What do you think?

**Comment by zamazan4ik** [comment]
So the valid example will be `encoding.avro.schema = "{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }"`

**Comment by jszwedko** [comment]
> So the valid example will be `encoding.avro.schema = "{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }"`

Hmm, yeah, I'm not actually very familiar with AVRO. Might need someone else to chime in 🤔 

At the least, it seems like the change here was what was intended in the docs so I'll merge it.

**Comment by sadathknorket** [comment]
Hi @zamazan4ik I followed the example and have the pipeline 

```
  sources:
    k8slogs:
      type: "kubernetes_logs"
      use_apiserver_cache: true


  transforms:
    remap_k8s:
      inputs:
        - 'k8slogs'
      type: 'remap'
      source: |
        if is_json!(.message) {
          structured = parse_json!(.message)
          . = merge!(.,structured)
          .pod_ip = .kubernetes.pod_ip
        }
        


  sinks:
    s3:
      type: aws_s3
      inputs:
        - remap_k8s
      auth:
        access_key_id: 
        secret_access_key: 
      compression: none  
      bucket: testavro17
      content_encoding: "avro"
      content_type: application/avro
      filename_extension: avro
      encoding:
        codec: avro
        avro:
          schema: "{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"pod_ip\", \"type\": \"string\" }] }"
      key_prefix: avro/
      region: eu-north-1
```

But it sends invalid avro files to s3 . Could you please help us out if your experienced with this

**Pull request merged**